Netatmo.lua

```diff
@@ -307,15 +318,33 @@ function QuickApp:getNetatmoDevicesData(token, mode)
     )
 end
 
+function QuickApp:addInterface(id, param)
+    local device = api.get('/devices/' .. tostring(id))
+    for _, interface in ipairs(device.interfaces) do
+        if interface == param then
+            self:debug("Add '" .. param .. "' interface to device #" .. tostring(device.id))
+            child:addInterfaces({param})
```

Comment:
addInterfaces method doesn't exists. Is it a Fibaro function?

Reply:
Yes addInterfaces() is a function provided by Fibaro, reported by @tinman on Fibaro forum : https://forum.fibaro.com/topic/49113-hc3-quickapps-coding-tips-and-tricks/page/8/